test/swift/EngineBuilderTests.swift

```diff
@@ -250,6 +250,20 @@ final class EngineBuilderTests: XCTestCase {
     self.waitForExpectations(timeout: 0.01)
   }
 
+  func testAddingH2ExtendKeepaliveTimeoutAddsToConfigurationWhenRunningEnvoy() {
+    let expectation = self.expectation(description: "Run called with enabled happy eyeballs")
```

Comment:
let's update the `description` in here

Reply:
Oops. Fixed in [e5c8020](https://github.com/envoyproxy/envoy-mobile/pull/2229/commits/e5c802075f2da12b2633d0e8d9dd5a8188a2fe2f)